packages/dm-core-plugins/src/yaml/YamlPlugin.tsx

```diff
@@ -58,7 +58,7 @@ const YamlView = (props: {
 
   return (
     <div>
-      <div className='flex justify-end items-end my-2 gap-1'>
+      <div className='flex justify-end items-center my-2 gap-1'>
```

Comment:
Doesn't this make it unaligned with the depth input field? 🤔 I seem to recall the reason I aligned it to the end was because the label on the input field made the centering not look nice.

Reply:
It was unaligned. Now its a bit better, still not perfect :/ 
The buttons did just change, your alignment might have been before that?

Reply:
aaah, probably